msg/json/synonyms.json

```diff
@@ -1,22 +1 @@
-{
```

Comment:
another suspiciously reformatted file... I think this is one that I manually reverted in https://github.com/google/blockly/pull/8162 so you may want to do the same thing

Reply:
Done